pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.554.3</version>
+    <version>1.580.1</version>
```

Comment:
I see. I also used that pattern unless someone (@jglick ?) suggested to use the latest LTS release  sometime ago. But maybe I'm not remembering correctly...